.github/workflows/upload-pypi.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish Python on TestPyPi
+name: Publish Python on PyPi
 
 on:
   push:
```

Comment:
would it be possible to have non-tagged versions pushed to test.pypi?

Reply:
By non-tagged versions, you mean all commits except the tagged one I imagine.
We could push them to test.pypi yes. The simpler would probably be to duplicate this file and to change the trigger in the new file. But that adds some maintenance (dealing with a duplicated file) and I don't see the usecase. Especially since you can do `python -m pip install git+https://github.com/bluebrain/PROJECT@HASH`

Actually, I'm not sure it would work since untagged version have a dirty version number. But maybe we can solve this by changing setuptools_scm behavior.

Also, testpypi changed its behavior, you can no longer (I don't know since when, I just noticed while working on the wheels) overwrite a version. So that would add some mess as well.

Something we can probably do though, is to upload the commit tarball to github artifacts, if needed. But again, I don't see any use case atm.

Reply:
ok, don't worry about it then